test/framework/flux.go

```diff
@@ -280,6 +281,57 @@ func (e *ClusterE2ETest) validateWorkerNodeMultiConfigUpdates(ctx context.Contex
 			return fmt.Errorf("machine template name should change on machine resource updates, old %s and new %s", machineTemplateName, newMachineTemplateName)
 		}
 		return nil
+	case v1alpha1.CloudStackDatacenterKind:
```

Comment:
I suggest not using this method anymore for flux upgrade test. This method is messy and not flexible. Instead, use `UpgradeWithGitOps` method when creating the tests.

reference: https://github.com/aws/eks-anywhere/blob/main/test/e2e/workload_clusters_test.go#L66

Reply:
Updated to use new method